flang/test/Lower/OpenMP/shared-loop.f90

```diff
@@ -1,9 +1,11 @@
 ! RUN: bbc -emit-hlfir -fopenmp %s -o - | FileCheck %s
 ! RUN: %flang_fc1 -emit-hlfir -fopenmp %s -o - | FileCheck %s
 
-! CHECK:  func.func @_QQmain() attributes
-! CHECK:    %[[ALLOC_I:.*]] = fir.alloca i32 {bindc_name = "i", uniq_name = "_QFEi"}
-! CHECK:    %[[DECL_I:.*]]:2 = hlfir.declare %[[ALLOC_I]] {uniq_name = "_QFEi"} :
+! --- Check that with shared(i) the variable outside the parallel section
+! --- is updated. 
+! CHECK:  func.func @_QPomploop()
```

Comment:
nit: use CHECK-LABEL for the function headers. It makes it easier to find test failures.